### ISSUE TITLE
Undeclared dependency

### ISSUE THREAD

**Issue by shaheerszm** [szymmis/vite-express#27]
With React+TS, when I run 'yarn dev' for the first time there is an error:

```
Error: vite-express tried to access picocolors, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: picocolors
```

**Comment by szymmis** [comment]
HI @shaheerszm, can you tell me if you installed `vite-express` as a dependency yourself or did you use `create-vite-express` package to setup the project? For me it looks like you don't have Vite installed as a dependency because I rely on it to have picocolors installed, that's why it is defined as a peer dependency instead of a dependency. We can rely on the Vite to have it because you need to have Vite installed either way to use the `vite-express` package. If you don't have it installed you can try installing Vite. If that doesn't help, please provide me some more info so I can help you. Thanks in advance!

**Comment by shaheerszm** [comment]
I used the command `yarn create vite-express` to setup the project. `yarn version 3.4.1` and `node version 18.9.0` in WSL2.0 (Ubuntu). I've used Vite to create other React+TS projects without issues before.
I don't know if this is relevant but I'm also using Yarn  @Plug'n'Play
@szymmis

**Comment by szymmis** [comment]
Thanks for your reply @shaheerszm. Info about using plug-n-play is very relevant because I think that it is a root cause why you have that problem and I don't. I've published fix in `v0.5.1`, can you check if the problem was solved or not?

**Comment by szymmis** [comment]
Should be fixed by #32.
